### ISSUE TITLE
Remote playback broken as of July 2015

### ISSUE THREAD

**Issue by mathrick** [learningit/plugin.video.vimeo#3]
Opening Vimeo URLs remotely via JSON-RPC has been broken since at least late May. It's the same issue that plagued the original unpatched plugin, at least going by the symptoms. Is there any chance of getting this fixed, and is there some kind of common thing that changes to result in this breakage? Knowing that would help with preparing patches in the future.

**Comment by learningit** [comment]
This version is not being updated. Please use the Bromix version in kodi.org repo

**Comment by mathrick** [comment]
If we're talking about the default one in upstream repos, then it has never worked properly (that's the one I referred to as "original unpatched"), and was the reason I switched to your version in the first place. I had to do version hackery to keep Kodi from updating, since upstream currently sits at 4.3.0 or thereabouts. I know it's not your fault that the upstream is broken, but I just want to clarify if we're talking about the same thing, because as far as I'm aware, there's no version anywhere that works.
